BSB_LAN/BSB_LAN.ino

```diff
@@ -529,62 +529,63 @@ EEPROMClass EEPROM_ESP("nvs", EEPROM_SIZE);
 #include <Wire.h>
 #include "src/BlueDot_BME280/BlueDot_BME280.h"
 BlueDot_BME280 *bme;  //Set 2 if you need two sensors.
+//Multiplexor TCA9548A (if presented) address on I2C bus
+#define TCA9548A_ADDR 0x70
 #endif
 
 bool client_flag = false;
 #ifdef WIFI
   #ifdef ESP32
-#include <WiFi.h>
-bool localAP = false;
-unsigned long localAPtimeout = millis();
+    #include <WiFi.h>
```

Comment:
Indention is wrong here. Code indention should be independent of compiler directive indention. So `bool localAP = false;` should be on the same level as `bool client_flag = false` etc.

Reply:
Now it looks better?

Reply:
Yes, thanks :)!